Source/monster.h

```diff
@@ -213,6 +219,26 @@ struct Monster { // note: missing field _mAFNum
 	CMonster *MType;
 	const MonsterData *MData;
 
+	/**
+	 * @brief Sets the current cell sprite to match the desired direction and animation sequence
+	 * @param graphic Animation sequence of interest
+	 * @param direction Desired direction the monster should be visually facing
+	*/
+	void ActivateAnimation(MonsterGraphic graphic, Direction direction)

```

Comment:
I would rename it to `ChangeAnimationData` to match the method in `AnimationInfo` (or if we dislike it, rename the version in `AnimationInfo`), cause both do more or less the same. Only `ChangeAnimationData` is even more safe (cause it regards if the number of frames differ from one animation to another).

(Same for `ActivateAnimationForCurrentDirection`)